src/SSHLibrary/config.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, **entries):
         self._config = entries
 
     def __str__(self):
-        return '\n'.join(['%s=%s' % (k, v) for k, v in self._config.items()])
+        return '\n'.join(['%s=%s' % (k, v) for k, v in list(self._config.items())])
```

Comment:
This change is not needed. Iterating over dictionary items works the same regardless the Python version. Also the existing pair of `[ ]` surrounding the expression is unnecessary, but don't think it should be removed in this PR.